apps/admin-server/src/pages/projects/[project]/widgets/map/[id]/index.tsx

```diff
@@ -16,8 +16,14 @@ import WidgetMapFilter from './filter';
 import WidgetMapReaction from './reaction';
 import { useRouter } from 'next/router';
 import Preview from '@/components/widget-preview';
+import { InferGetServerSidePropsType } from 'next';
+import { withApiUrl } from '@/lib/server-side-props-definition';
+import WidgetPublish from '@/components/widget-publish';
+export const getServerSideProps = withApiUrl;
 
-export default function WidgetMap() {
+export default function WidgetMap({
+  apiUrl,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) {
```

Comment:
Deze gebruikt een ander type, werkt ook natuurlijk maar is denk netter om dit gelijk te trekken?